### PR TITLE
Remove the gap in search space value array

### PR DESCRIPTION
**Pull request by lvybriage**

### CODE REVIEW

File: src/webui/src/components/Sessionpro.tsx [hunk 1]
```diff
@@ -138,16 +138,28 @@ class Sessionpro extends React.Component<{}, SessionState> {
...
+                        let value = searchSpace[item]._value;
+                        switch (key) {
+                            case 'loguniform':
+                            case 'qloguniform':
+                                const a = Math.pow(10, value[0]);
+                                const b = Math.pow(10, value[1]);
+                                if (a < b) {
+                                    value = [a, b];
+                                } else {
+                                    value = [b, a];
```
Comment (chicm-ms) [inline src/webui/src/components/Sessionpro.tsx]: Maybe we should not change the order of the values specified by user in search space?

Comment (lvybriage) [reply src/webui/src/components/Sessionpro.tsx]: ok

File: src/webui/src/components/Sessionpro.tsx [hunk 2]
```diff
@@ -138,16 +138,24 @@ class Sessionpro extends React.Component<{}, SessionState> {
...
-                                value = [b, a];
-                            }
-                            searchSpace[item]._value = value;
+                                searchSpace[item]._value = value;
+                                break;
+
+                            case 'quniform':
+                            case 'qnormal':
+                            case 'qlognormal':
+                                searchSpace[item]._value = [value[0], value[1]];
```
Comment (chicm-ms) [inline src/webui/src/components/Sessionpro.tsx]: This line (154) seems does nothing?

Comment (chicm-ms) [reply src/webui/src/components/Sessionpro.tsx]: Ok, got it.

**Pull request merged**